examples/api/lib/widgets/pop_scope/pop_scope.1.dart

```diff
@@ -0,0 +1,233 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This sample demonstrates how to use a PopScope to wrap widget that
```

Comment:
Missing "a" in this new sample.

```suggestion
// This sample demonstrates how to use a PopScope to wrap a widget that
```